app/views/posts/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Here is a list of all posts</h1>
-
+<% @posts.each do |post| %>
+  <%= link_to user_post_path(user_id: @user.id, id: post.id, author_id: post.id) do %>
+    <h1><%= post.title %></h1>
+    <p><%= post.text %></p>
+    <% if post.recent_comments.empty? %>
+      <p>No comments</p>
+    <% else %>
+    <% post.recent_comments.each do |comment| %>
+  <p><%= comment.author.name %>: <%= comment.text %></p>
+<% end %>
+    <% end %>
+  <% end %>
+<% end %>
```

Comment:
- [ ] [REQUIRED] Let's also ensure that when we click the **See all Post** button, it should open the user's post index as displayed on the wireframe.

![](https://...image1.png) | ![](https://...image2.png)
Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="483" alt="Screenshot 2023-06-17 at 20 10 54" src="https://github.com/Lornakaboro/Rails-blog/assets/58692590/090815c6-847f-452f-89fb-d2dc66535697"> | "<img width="485" alt="Screenshot 2023-06-17 at 19 54 28" src="https://github.com/Lornakaboro/Rails-blog/assets/58692590/095b5b8e-a19c-428a-ba47-299f327f4ab9">">